roadmap/implementors-guide/guide.md

```diff
@@ -513,7 +516,11 @@ PastCode: map (ParaId, BlockNumber) => Option<ValidationCode>;
 /// but we also keep their code on-chain for the same amount of time as outdated code
 /// to keep it available for secondary checkers.
 PastCodeMeta: map ParaId => ParaPastCodeMeta;
-/// Which paras have past code that needs pruning and the relay-chain block in which context the code was replaced.
+/// Which paras have past code that needs pruning and the relay-chain block at which the code was replaced.
+/// Note that this is the actual height of the included block, not the expected height at which the
+/// code upgrade would be applied, although they may be equal.
+/// This is to ensure the entire acceptance period is covered, not an offset acceptance period starting
+/// from the time at which the parachain perceives a code upgrade as having occurred.
```

Comment:
@burdges I updated the docs here to reflect a change in semantics around the meaning of the block number in `PastCodePruning`.

Here by included I truly mean backed+available, as the changes concern keeping code available for an entire acceptance period.